TESTS/mbed_hal/stack_size_unification/main.cpp

```diff
@@ -37,10 +37,22 @@ extern uint32_t mbed_stack_isr_size;
 
 #define EXPECTED_USER_THREAD_DEFAULT_STACK_SIZE  (MBED_CONF_RTOS_THREAD_STACK_SIZE)
 
-#if ((MBED_RAM_SIZE - MBED_BOOT_STACK_SIZE) <= (EXPECTED_MAIN_THREAD_STACK_SIZE + EXPECTED_ISR_STACK_SIZE))
+#if ((MBED_RAM_SIZE - MBED_CONF_TARGET_BOOT_STACK_SIZE) <= (EXPECTED_MAIN_THREAD_STACK_SIZE + EXPECTED_ISR_STACK_SIZE))
 #error [NOT_SUPPORTED] Insufficient stack for staci_size_unification tests
 #endif
 
+/* Workaround a bug where the boot stack size configuration option is not
```

Comment:
Instead of introducing a workaround, shouldn't we correct #13474 first ...?

Reply:
The legacy CLI tools are frozen and can't be fixed, so a workaround is necessary unfortunately.